build/package/rpm/SPECS/nginx-asg-sync.spec

```diff
@@ -3,7 +3,7 @@
 
 Summary: NGINX Plus integration with AWS Auto Scaling groups
 Name: nginx-asg-sync
-Version: 0.1
+Version: 0.2
 Release: 2%{?dist}
```

Comment:
Please change `Release: 2%{?dist}` to `Release: 1%{?dist}`
as the current value produces `nginx-asg-sync-0.2-2.el7.x86_64.rpm` instead of `nginx-asg-sync-0.2-1.el7.x86_64.rpm` same for the package for Amazon linux